mozilla/blurts-server
# fix primary email breaches initially hidden

# References: 
Jira: MNTOR-1164



# Description
Fixes the bug where upon initial load the breaches page does not show breaches for a primary email account.

# How to test

1. create a new Monitor account with primary email that has known breaches
2. visit the breaches page and notice the breaches are not visible


# Checklist (Definition of Done)
